src/main/java/fi/otavanopisto/kuntaapi/server/index/IndexableService.java

```diff
@@ -42,81 +97,239 @@ public String getType() {
   public Long getOrderIndex() {
     return orderIndex;
   }
-  
+
   public void setOrderIndex(Long orderIndex) {
     this.orderIndex = orderIndex;
   }
-  
+
   public String getServiceId() {
     return serviceId;
   }
 
   public void setServiceId(String serviceId) {
     this.serviceId = serviceId;
   }
-  
+
   public List<String> getOrganizationIds() {
     return organizationIds;
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Add the missing @deprecated Javadoc tag. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AMissingDeprecatedCheck)
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Do not forget to remove this deprecated code someday. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1133)